searchapp/static/student_view.js

```diff
@@ -188,22 +188,207 @@ function upload_click(e) {
 			data: formData,
 			headers: { "X-CSRFToken": csrftoken, crossOrigin: false},
 			success: function(d) {
+				$('#paper-chapter').dropdown('clear');
+				$('#paper-subject-splits').dropdown('clear');
 				$(`#${worksheetType}-chapter-options-parent`).empty();
+				$(`#paper-subject-splits-options-parent`).empty();
 				let chapters = d['chapters'];
-				$(`#${worksheetType}-chapter`).removeClass('hide-display').addClass('show-display');
+				let subject_breakup = d['subject_breakup'];
+				console.log(subject_breakup);
+				$(`#subject_splits`).removeClass('show-display').addClass('hide-display');
+				$('h3').removeClass('show-display').addClass('hide-display');
+				$(`#chapter-operations`).removeClass('hide-display').addClass('show-display');
+				$('#add-subject-split').removeClass('hide-display').addClass('show-display');
 				for (var i=0; i<chapters.length; i++) {
 					$(`#${worksheetType}-chapter-options-parent`).append(`<div class="item" data-value=${chapters[i]['chapter_id']}>${chapters[i]['chapter_name']}</div>`);
 				}
+				for (var i=0; i<subject_breakup.length; i++) {
+					$(`#paper-subject-splits-options-parent`).append(`<div class="item" data-value=${subject_breakup[i]['breakup_name']}>${subject_breakup[i]['breakup_name']}</div>`);
+				}
 			}
 		});
 	}
 
+	function add_chapter(e)
```

Comment:
Here as well. Give your parameters meaningful names. Ideally the code should document itself.